### DISCUSSION TITLE
When installing a beta version should I uninstall my previous version?

### DISCUSSION THREAD

**Discussion by IsmaelPR1977** [OrcaSlicer/OrcaSlicer#4484] [Q&A]
This may seem like a silly question but if I am installing a beta version should I uninstall my previous stable version? My thinking says to keep the old one but I wasn't sure of the best practice. Thanks in advance.

**Comment by LJKraus** [comment]
It's your choice, but I do not.  New version replaces old; if I don't like it, I re-install the older release. Another option is to just install the portable beta version in a different folder and then run the beta only when you want to experiment.  When beta testing is over, install the stable release and delete the portable folder.
